assets/js/klp-payment.js

```diff
@@ -26,6 +27,27 @@ const payload = {
     }
 }
 
+if (klp_payment_params.firstname) {
+    payload.data.first_name = klp_payment_params.firstname;
+}
+
+if (klp_payment_params.lastname) {
+    payload.data.last_name = klp_payment_params.lastname;
+}
+
+if (klp_payment_params.email) {
+    payload.data.email = klp_payment_params.email;
+}
+
+if (klp_payment_params.phone) {
+    if (klp_payment_params.phone.length > 11) {
```

Comment:
Curious, why are we testing to see if number exceed 11 numbers? Standard number is 11 number.

Reply:
If a user phone number is +234703.... , that's more than 11.
It throws an error. I tested for that case and had to check for it. @cyberomin